bindata/v4.0.0/controller/role.yaml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: system:openshift:controller:service-ca
   namespace: openshift-service-ca
 rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - restricted
+  verbs:
+  - use
```

Comment:
"use" is a verb?

Reply:
yes, it's used in the SCC admission: https://github.com/openshift/origin/blob/3c26fadaf29adfeefab38907792ebb6167526d3c/vendor/github.com/openshift/apiserver-library-go/pkg/securitycontextconstraints/sccmatching/matcher.go#L74-L74